src/main/java/carRacing/car/Cars.java

```diff
@@ -0,0 +1,31 @@
+package carRacing.car;
+
+import carRacing.moveStrategy.MoveStrategy;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+	private List<Car> cars = new ArrayList<>();
+
+	public Cars(List<String> carNames) {
+		makeCars(carNames);
+	}
+
+	private List<Car> makeCars(List<String> carNames) {
+		for (int i = 0; i < carNames.size(); i++) {
+			cars.add(new Car(0, carNames.get(i)));
+		}
+		return cars;
```

Comment:
```suggestion
	public Cars(List<String> carNames) {
		makeCars(carNames);
	}

	private void makeCars(List<String> carNames) {
                for (String carName : carNames) {
                    cars.add(new Car(carName));
                }
	}
```
리스트 순회를 직접 돌리는 방식보다 for-each 문법을 사용하는 것이 깔끔합니다.
반환할 필요도 없으니 void로 변경하는 것도 맞겠네요